src/logic/api/index.js

```diff
@@ -0,0 +1,75 @@
+import axios from 'axios';
+
+const defaultConfig = {
+  baseUrl: `${process.env.REACT_APP_API_URL}${process.env.REACT_APP_API_PATH_PREFIX}`,
+  timeout: 10_000,
+  headers: {
+    Accept: 'application/json',
+    'Content-Type': 'application/json',
+  },
+};
+
+const api = axios.create(defaultConfig);
+
+const ownInterseptors = {
+  handleRequestError: error => {
+    return Promise.reject(error);
+  },
+  handleResponseError: error => {
+    return Promise.reject(error);
+  },
+  handleResponse: response => {
+    return response;
+  },
+  handleRequest: config => {
+    return config;
+  },
+};
+
+api.interceptors.request.use(ownInterseptors.handleRequest, ownInterseptors.handleRequestError);
+
+api.interceptors.response.use(ownInterseptors.handleResponse, ownInterseptors.handleResponseError);
+
+const Api = {
+  API_CALL: 'API_CALL',
+
+  catchError: error => {
+    return error.response.data;
+  },
+
+  delete: path => {
```

Comment:
For each crud method add a new parameter called config and initialise it with `defaultConfig`, then, where you call the method add config as the last argument.

**Notice the difference between `argument` and `parameter`** -- look it up on Google